client.js

```diff
@@ -1,24 +1,29 @@
 const crypto = require('crypto');
 const https = require('https');
+require('dotenv').config();
 
-const apiKey = "add api key here";
-const privateKey = "add private key here";
+const apiKey = process.env.API_KEY;
+const privateKey = process.env.PRIVATE_KEY;
```

Comment:
I would suggest using nconf over dotenv, it's far more "12 factor" and a better user experience.

```suggestion
const ENV_SEPARATOR = '__';
const KEY = 'btcmarkets';
const ENV_PATTERN = new RegExp(`/^${KEY.toUpperCase()}${ENV_SEPARATOR}/`);

nconf
  .env({
    separator: ENV_SEPARATOR,
    match: ENV_PATTERN,
    lowerCase: true,
    parseValues: true,
    transform(obj) {
      obj.key.replace(ENV_PATTERN, '');
      return obj;
    },
  })
  .argv({
    apiKey: {
      describe: 'prefix of all new packages created',
    },
    privateKey: {
      describe: 'prefix of all new packages created',
    }
  });
  
  const apiKey = nconf.get('apiKey');
  const privateKey = nconf.get('privateKey');
```

Now the cli is used like this: 

```shell
node client.js --apiKey=something --privateKey=another-thing
```

or 

```shell
BTCMARKETS_API_KEY=something \
BTCMARKETS_API_PRIVATE=another-thing \
node client.js 
```

you'll also get : 

```shell
node client.js --help
```